ktor-server/ktor-server-tests/jvmAndNix/test/io/ktor/tests/server/routing/RoutingProcessingTest.kt

```diff
@@ -887,6 +887,29 @@ class RoutingProcessingTest {
         }
     }
 
+    @Test
+    fun testRoutingSpecificErrorStatusCodeOnlyForConstantQualityPath() = testApplication {
```

Comment:
Could you also add a test with a prefix(like `/api/v1/`)?

Reply:
There are other tests in this class that test route with complex path responding with 405 status code